### PR TITLE
ID edits

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,25 @@
...
 
-- **[Documentation](https://microclimate.dev/codewindtechpreview)**
-- **[Changelog](https://github.com/microclimate-dev2ops/codewind-vscode/blob/master/CHANGELOG.md)**
-
-Create and develop cloud-native, containerized web applications from VS Code.
-
-## Getting started
+## Installing Codewind
+Prerequisites
+- Install [VS Code](https://code.visualstudio.com/download).
```
Comment (sishida) [inline README.md]: Add Docker and Docker Compose as prereqs.

Comment (sishida) [reply README.md]: Docker Compose is already included with Docker on a Mac.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,25 @@
...
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+Complete all parts of the Codewind installation:
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions). Then, click the **Install** button.
+2. Go to the **Explorer** view group and open the **Codewind** view.
+3. To complete the installation, open the Codewind view in the **Explorer** view group or type `Focus on Codewind` into the Command Palette. A notification appears and asks if you would like to finish installing Codewind. 
+4. Click **Install** to complete the Codewind installation. The file is <size of download>. For more information, see <more info link> (same link as in pop up). When the installation completes, the server automatically starts.
```
Comment (sishida) [inline README.md]: The file size is approximately 1 GB.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,25 @@
-<!--a href="https://microclimate-dev2ops.github.io/">
-    <img src="mc-banner.png" alt="Microclimate Banner" height="75px" />
-</a-->
-
 # Codewind for VS Code
+Create and develop cloud-native, containerized web applications from VS Code.
 
 [![Marketplace](https://img.shields.io/vscode-marketplace/v/IBM.codewind.svg?label=marketplace&logo=visual-studio-code)](https://marketplace.visualstudio.com/items?itemName=IBM.codewind)
 [![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
 [![Slack](https://img.shields.io/badge/ibm--cloud--tech-blue.svg?logo=slack&label=slack)](https://slack-invite-ibm-cloud-tech.mybluemix.net/)
```
Comment (sishida) [inline README.md]: Move badges to top or right under top header.

File: README.md (hunk 2 above)
Comment (sishida) [reply README.md]: Delete the "more info" link.

File: README.md [hunk 4]
```diff
@@ -1,31 +1,27 @@
...
-
+# Codewind for VS Code
 Create and develop cloud-native, containerized web applications from VS Code.
 
-## Getting started
+## Installing Codewind
+Prerequisites
+- Install [VS Code](https://code.visualstudio.com/download).
+- Install Docker.
+- If you use Windows, you also need to install Docker Compose.
```
Comment (tetchel) [inline README.md]: windows or linux

File: README.md [hunk 5]
```diff
@@ -1,31 +1,27 @@
...
 
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+Complete all parts of the Codewind installation:
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions). Then, click the **Install** button.
+2. Go to the **Explorer** view group and open the **Codewind** view.
+3. To complete the installation, open the Codewind view in the **Explorer** view group or type `Focus on Codewind` into the Command Palette. A notification appears and asks if you would like to finish installing Codewind.
```
Comment (tetchel) [inline README.md]: I still think the "a notification appears" should be removed

File: README.md [hunk 6]
```diff
@@ -1,31 +1,27 @@
...
 
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+Complete all parts of the Codewind installation:
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions). Then, click the **Install** button.
+2. Go to the **Explorer** view group and open the **Codewind** view.
+3. To complete the installation, open the Codewind view in the **Explorer** view group or type `Focus on Codewind` into the Command Palette. A notification appears and asks if you would like to finish installing Codewind. 
+4. Click **Install** to complete the Codewind installation. The file is approximately 1 GB. When the installation completes, the server automatically starts.
```
Comment (tetchel) [inline README.md]: > The file is approximately 1 GB
it's not a file persay. it's a set of docker images. perhaps "The download is approximately 1 GB"

File: README.md (hunk 5 above)
Comment (sishida) [reply README.md]: If we remove the notification, how should we inform the users that the pop-up is normal? This was a concern brought up in the UI/Design call.

Comment (tetchel) [reply README.md]: most users won't even look at the readme, the notification is responsible for making itself seem normal

Comment (tetchel) [reply README.md]: and I think that that is quite easy to achieve. "more needs to be installed" is unusual but won't be shocking to a user.

File: README.md (hunk 4 above)
Comment (tetchel) [reply README.md]: technically "if you are not on macOS" you also need to install docker-compose

Comment (tetchel) [reply README.md]: Sorry I'm wrong, it's actually *only* linux that doesn't include docker-compose ootb

File: README.md [hunk 7]
```diff
@@ -1,31 +1,27 @@
...
+- Install Docker.
+- If you use Linux, you also need to install Docker Compose.
 
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+Complete all parts of the Codewind installation:
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind) or by searching for `Codewind` in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions). Then, click the **Install** button.
+2. Go to the **Explorer** view group and open the **Codewind** view.
+3. To complete the installation, more needs to be installed. Open the Codewind view in the **Explorer** view group or type `Focus on Codewind` into the Command Palette.
+4. Click **Install** to complete the Codewind installation. The download is approximately 1 GB.
```
Comment (tetchel) [inline README.md]: how about:

Click **Install** *when prompted* to complete the Codewind installation

**Comment by sishida** [comment]
@tetchel I'm not seeing an option to merge the PR. Do we need to request a committer to merge for us?

**Comment by tetchel** [comment]
yes, someone will do it

**Pull request merged**